apps/explorer/src/app/routes/parties/home/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const Parties = () => {
+  return (
+    <section>
+      <h1>Parties</h1>
+      <h2>
+        Not sure what to do with this page? Could show all parties but would
```

Comment:
Would it make sense for this page to have a search box to find a party and then show all TX for that party? Probably a follow up feature though!